src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -40,6 +49,10 @@ import {
   SkyTabsetService
 } from './tabset.service';
 
+interface SkyTabsetPermalinkParams {
```

Comment:
Is there a reason that we are putting this interface here and not in a file in a `types` folder like we do in most of our repos? I tend to discourage adding more than once class/interface/etc in a single file.

Reply:
It's only used by this file, but I can pull it out for the sake of consistency.

Reply:
Thanks. I tend to personally just pull them out to make them easier to find down the road.